geniza/footnotes/migrations/0016_rename_typed_texts.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.2.8 on 2022-02-10 18:12
+
+from django.conf import settings
+from django.db import migrations
+
+
+def rename_typed_texts(apps, schema_editor):
+    Source = apps.get_model("footnotes", "Source")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    LogEntry = apps.get_model("admin", "LogEntry")
+    source_contenttype = ContentType.objects.get(app_label="footnotes", model="source")
+    User = apps.get_model("auth", "User")
+    (user, _) = User.objects.get_or_create(
+        username=settings.SCRIPT_USERNAME, is_staff=False, is_active=False
+    )  # generic user for script execution (should already be created)
+
+    # get all sources called "typed texts"
+    typed_texts = Source.objects.filter(title="typed texts")
+    for source in typed_texts:
+        # rename to "unpublished editions"
+        source.title = "unpublished editions"
+        source.title_en = "unpublished editions"
+        source.save()
+        # log action
+        LogEntry.objects.log_action(
+            user_id=user.id,
+            content_type_id=source_contenttype.pk,
+            object_id=source.pk,
+            object_repr=str(source),
+            change_message='changed title "typed texts" to "unpublished editions"',
+            action_flag=2,  # CHANGE
```

Comment:
can we not import the constant not available in migrations?

Reply:
I wasn't sure but it worked fine! Done in a80faf6 👍